fbw-a32nx/src/systems/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx

```diff
@@ -141,72 +170,72 @@ export const FlightWidget = () => {
                     </h1>
                 )}
             </div>
-            <div className="overflow-hidden relative p-6 w-full rounded-lg border-2 h-content-section-reduced border-theme-accent">
-                <div className="flex flex-col justify-between h-full">
+            <div className="h-content-section-reduced border-theme-accent relative w-full overflow-hidden rounded-lg border-2 p-6">
+                <div className="flex h-full flex-col justify-between">
```

Comment:
May be better not to commit linting changes like this, maybe remove from the diff for this PR and at some point there should be a PR to clean this up.

Reply:
Well, I can't stop that from happening unless I change the provided vscode Settings. Wasn't an Issue on PR 8407.
(Not that I would like going over Diffs on Areas that I have not changed^^)